nvpy/view.py

```diff
@@ -1448,12 +1478,15 @@ def _create_ui(self):
             self.notes_list.pack(fill=tk.BOTH, expand=1)
 
             sort_mode_frame = tk.Frame(list_frame)
-            sort_mode_frame.pack(side=tk.TOP, fill=tk.X)
+            if not self.config.streamline_interface:
+                sort_mode_frame.pack(side=tk.TOP, fill=tk.X)
             sort_mode_label = tk.Label(sort_mode_frame, text='Sort by')
-            sort_mode_label.pack(side=tk.LEFT)
+            if not self.config.streamline_interface:
+                sort_mode_label.pack(side=tk.LEFT)
```

Comment:
Perhaps we can reduce if statements.
Just skip calling `parent_frame.pack()`. Child widgets can call `pack()`.

Reply:
Fixed by a0ff025e3afbccd067f605bdd39dfa21cb8e3cb3